index.html

```diff
@@ -266,5 +266,32 @@ <h3 class="skillsTitle">Skills</h3>
   </div>
 </section>
 
+            <!-- This is the contact form -->
+
+<section class="contact" id="contactForm">
+    <h2 class="contactHeader">Contact me</h2>
+    <p class="contactDescription">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+    <form action="https://formspree.io/f/mgedkave" class="contactForm" method="post">
+      <ul class="contactField">
+        <li>
+          <label for="name"></label>
+          <input class = "contactName" type="text" id="name" name="name" placeholder="Enter your name here ..." maxlength="30" required />
+        </li>
+        <li>
+          <label for="email"></label>
+          <input class="emailAddress" type="email" id="email" name="email" placeholder="Enter your e-mail here ..." required />
+        </li>
+        <li>
+          <label for="message"></label>
+          <textarea class="contactMessage" id="message" placeholder="Type your message here ..." maxlength="500" required></textarea>
+        </li>
+        <li class="formSubmit">
+          <input type="submit" class="submitButton" value="Get In Touch" />
+        </li>
+      </ul>
+    </form>
+</section>
```

Comment:
Your design             |  Figma Design
:-------------------------:|:-------------------------:
![ffff](https://user-images.githubusercontent.com/69638013/154088343-d30c49c1-05a4-40a7-b498-fe65b62566d7.PNG)  | ![contact form](https://user-images.githubusercontent.com/69638013/154084122-0f30f544-44ff-41d8-adba-00b926a900b8.PNG)  

- [ ] Kindly style placeholder text to make your design look like the original design
- [ ] Kindly add background images to the section this will make your design look like the original design
- You can wrap this section in a footer tag from the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) your contact form is in the footer  👍🏿